oci/private/pull.bzl

```diff
@@ -358,7 +359,12 @@ def _oci_pull_impl(rctx):
 
     mf, mf_len = downloader.download_manifest(rctx.attr.identifier, "manifest.json")
 
-    if mf["mediaType"] in _SUPPORTED_MEDIA_TYPES["manifest"]:
+    if "mediaType" in mf:
+        media_type = mf["mediaType"]
+    else:
+        media_type = mf["config"]["mediaType"]
```

Comment:
it feels like this now needs a final else clause when the media_type couldn't be found in either of these two paths within the data structure